docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+
+Local Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``~/.cumulusci/project_name/cumulusci.yml``
+
+**Windows:** ``%homepath%\.cumulusci\project_name\cumulusci.yml``
+
+Configurations made to this ``cumulusci.yml`` file apply solely to the project with the given <project_name>, and take precedence over **all other** configuration scopes except the universal ``cumulusci.yml`` file. If you want to make customizations to a project, but don't need them to be available to other team members, make those customizations here.
+
+
+Global Configurations
+^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``~/.cumulusci/cumulusci.yml``
+
+**Windows:** ``%homepath%\.cumulusci\cumulusci.yml``
+
+Configuration of this file overrides behavior across **all** CumulusCI projects on your machine. Configurations in this file have a low precedence, and are overridden by **all other** configurations except for those that are in the universal ``cumulusci.yml`` file.
+
+
+Universal Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is one more configuration file that exists: the `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file that ships with CumulusCI itself. This file actually holds the lowest precedence of all, as all other scopes override what is contained in this file. That said, it contains all of the definitions for the standard tasks, flows, and org configurations that come standard with CumulusCI.
+
+The commands ``cci task info`` and ``cci flow info`` display all of the infomation regarding how a particular task or flow is configured. They display both the information present in the standard library alongside any customizations that have been added.
+
+
+
+Advanced Configurations
+-----------------------
+
+
+Use Tasks and Flows From a Different Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It's also possible to use tasks and flows from another project with CumulusCI. To do this, the other project must be named under the ``sources`` section of the project ``cumulusci.yml`` file:
+
+    Example: When tasks or flows are referenced using the `npsp` namespace, CumulusCI fetches the source from the NPSP GitHub repository.
+
+.. code-block:: yaml
+
+    sources:
+      npsp:
+        github: https://github.com/SalesforceFoundation/NPSP
+
+By default, CumulusCI fetches the most recent release, or the default branch if there are no releases.
+
+.. note::
+    In order for this feature to work, the referenced repository needs to be readable (for example, it's public, or CumulusCI's GitHub service is configured with the token of a user who has read access to it).
+
+It's also possible to fetch a specific ``tag``:
```

Comment:
Given how you've structured the "or" part of the sentence below (which I like), consider ending with an ellipsis here to prepare the reader for the excitement of more to come after the code snippet?